docs/api/v1.24.md

```diff
@@ -1659,6 +1659,7 @@ Build an image from a Dockerfile
 **Example request**:
 
     POST /v1.24/build HTTP/1.1
+    Content-Type: application/tar
```

Comment:
hm, interesting. docs look good then, but wondering why we are inconsistent there